CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Changed
 - [#28](https://github.com/seznam/slo-exporter/pull/28) Abstracted statistical classifier history to generic interface in new storage package.
 
+## Fixed
+- [#33](https://github.com/seznam/slo-exporter/pull/33) prometheus-ingester now always expose query_fails_total
+
```

Comment:
side note: we are missing the previous changes (new grafana dashboards, version of other grafana dashboards, refactor of prometheus_rules and add of rate coefficient).

Reply:
Fair point, thank you.

I will add those when I will be cutting a new release (as soon as I will merge this PR).